src/datagen/generated/minecolonies/data/minecolonies/tags/items/dyed_terracotta.json

```diff
@@ -0,0 +1,21 @@
+{
+  "replace": false,
+  "values": [
+    "minecraft:white_terracotta",
+    "minecraft:orange_terracotta",
+    "minecraft:magenta_terracotta",
+    "minecraft:light_blue_terracotta",
+    "minecraft:yellow_terracotta",
+    "minecraft:lime_terracotta",
+    "minecraft:pink_terracotta",
+    "minecraft:gray_terracotta",
+    "minecraft:light_gray_terracotta",
+    "minecraft:cyan_terracotta",
+    "minecraft:purple_terracotta",
+    "minecraft:blue_terracotta",
+    "minecraft:brown_terracotta",
+    "minecraft:green_terracotta",
+    "minecraft:red_terracotta",
+    "minecraft:black_terracotta"
```

Comment:
I think DO has a tag for terracotta

Reply:
Actually it looks like vanilla MC does in 1.18... they didn't in 1.16 though.  I'll change that over.